<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote IR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../assets/icons/app-icon.svg</normaloff>../../assets/icons/app-icon.svg</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="updatePortListBtn">
        <property name="toolTip">
         <string>Refresh port list</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../assets/icons/sync-solid.svg</normaloff>../../assets/icons/sync-solid.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialPortCmbx">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select port</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serialRateLe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Serial port speed</string>
        </property>
        <property name="placeholderText">
         <string>Serial port rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="receiveChangeBtn">
        <property name="toolTip">
         <string>Start/Stop port listening</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../assets/icons/play-solid.svg</normaloff>../../assets/icons/play-solid.svg</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="infoBtn">
        <property name="toolTip">
         <string>About programm</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../assets/icons/info-solid.svg</normaloff>../../assets/icons/info-solid.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Bound signals:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="signalLw">
          <property name="toolTip">
           <string>All processed signals</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="addCommandBtn">
            <property name="toolTip">
             <string>Add command</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../assets/icons/plus-solid.svg</normaloff>../../assets/icons/plus-solid.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveCommandBtn">
            <property name="toolTip">
             <string>Save selected command</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../assets/icons/save-solid.svg</normaloff>../../assets/icons/save-solid.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteCommandBtn">
            <property name="toolTip">
             <string>Delete selected command</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../assets/icons/trash-solid.svg</normaloff>../../assets/icons/trash-solid.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
        <item>
         <layout class="QVBoxLayout" name="selectedCommandLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Selected signal:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Code</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="signalCodeLe">
              <property name="toolTip">
               <string>Signal code from port</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Select action</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="actionSelectCmbx">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Available actions</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>log:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="logTe">
            <property name="toolTip">
             <string>Actions log</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="updatePortListAct">
   <property name="text">
    <string>update port list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
